Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -505,6 +510,21 @@ extension NIOHTTP2Handler {
     }
 }
 
+extension NIOHTTP2Handler.ConnectionDemultiplexer {
+    func process(event: InboundEventBuffer.BufferedHTTP2UserEvent) {
+        switch event {
+        case .streamCreated(let event):
+            self.streamCreated(event.streamID, localInitialWindowSize: event.localInitialWindowSize, remoteInitialWindowSize: event.remoteInitialWindowSize)
+        case .initialStreamWindowChanged(let event):
+            self.initialStreamWindowChanged(by: event.delta)
+        case .streamWindowUpdated(let event):
+            self.streamWindowUpdated(event.streamID, inboundWindowSize: event.inboundWindowSize, outboundWindowSize: event.outboundWindowSize)
+        case .streamClosed(let event):
+            self.streamClosed(event.streamID, reason: event.reason)
```

Comment:
Seeing it like this makes me ask the question: is there a good reason these functions aren't defined to simply take the event directly?

Reply:
When I sketched out the API I didn't consider what was already there (i.e. the inbound event buffer). So no, no good reason to not use events directly.

Reply:
Changed to use the events directly.